include/roboteam_ai/analysis/PlaysObjects/Invariants/AlwaysTrueInvariant.h

```diff
@@ -0,0 +1,32 @@
+//
+// Created by jessevw on 11.12.19.
+//
+
+#ifndef RTT_ALWAYSTRUEINVARIANT_H
+#define RTT_ALWAYSTRUEINVARIANT_H
+
+#include "analysis/PlaysObjects/Invariants/Invariant.h"
+
+namespace rtt::ai::analysis {
+    /**
+     * Invariant that is true when the ball belongs to us
+     */
+    class AlwaysTrueInvariant : public Invariant {
+    private:
+        std::string name;
+    public:
+        /**
+         * Functional implementation of when the ball belongs to us
+         * @param world the current world state
+         * @param field the current field state
+         * @return true if the ball belongs to us, false otherwise
+         */
+        static bool isValid(rtt::ai::world::World *world, rtt::ai::world::Field *field);
```

Comment:
What's the point of having a constructor when your isValid function is static, it's literally a standalone function, why even bother with inheritance at that point? Just have an
std::vector<Invariant> at that point where Invariant is like:

```cpp
class Invariant {
    std::string name;
    std::function<bool(rtt::ai::world::World* world, Field* field)> callable;
};

```